packages/javascript/src/span.ts

```diff
@@ -172,14 +172,23 @@ function extractPath(backtraceLine: string): string | undefined {
   // preceded by space characters and followed by one space.
   const IS_CHROME = /^\s*at\s/
   // In a Chrome backtrace line, the path (if it is available)
-  // is located, usually within parentheses, after the "@" towards the
-  // end of the line, along with the line number and column number,
-  // separated by colons. We check for those to reject clear non-paths.
-  const CHROME_PATH = /at(?:\s.*)?\s\(?(.*):\d*:\d*\)?$/i
+  // is located after the "at" towards the end of the line, along with
+  // the line number and column number, separated by colons.
+  // When the function name is available, the path is enclosed in
+  // parentheses. When there is no function name, the path comes
+  // immediately after the "at".
+  // We check for the line and column numbers to reject clear non-paths.
+  const CHROME_PATH_WITH_FUNCTION_NAME = /^\s*at(?:\s[^\(]+)?\s\((.*):\d*:\d*\)$/
+  const CHROME_PATH_WITHOUT_FUNCTION_NAME = /^\s*at\s(.*):\d*:\d*$/
 
   if (backtraceLine.match(IS_CHROME)) {
-    const match = backtraceLine.match(CHROME_PATH)
-    return match ? match[1] : undefined
+    const match_with = backtraceLine.match(CHROME_PATH_WITH_FUNCTION_NAME)
+    if (match_with) {
+      return match_with[1]
+    }
+
+    const match_without = backtraceLine.match(CHROME_PATH_WITHOUT_FUNCTION_NAME)
+    return match_without ? match_without[1] : undefined
```

Comment:
I think there's a more TypeScript way to write this, like my suggestion, or something with `??`.

```suggestion
    return match_without?.[1]
```